nrf_rpc: Add changelog file. (#840)
nrfconnect/sdk-nrfxlib: nrf_rpc/CHANGELOG.rst

.. _nrf_rpc_changelog:

Changelog
#########

.. contents::
   :local:
   :depth: 2

All notable changes to this project in the |NCS| are documented in this file.

Main branch
***********

All the notable changes added to the |NCS| main are documented in this section.

Added
=====

> * Added configurations number of ZCBOR backups.

Reviewer: A bit unclear what is "configurations number", needs to be clarified. ZCBOR -> zcbor.

Author: ACK. Changed to: _Added :kconfig:option:CONFIG_NRF_RPC_ZCBOR_BACKUPS to configure number of zcbor backups._

> * Added :kconfig:option:CONFIG_NRF_RPC_ZCBOR_BACKUPS to configure number of zcbor backups.

Reviewer: ```suggestion
* Added the :kconfig:option:`CONFIG_NRF_RPC_ZCBOR_BACKUPS` Kconfig option for setting up the number of zcbor backups.
```

nRF Connect SDK v2.1.0
**********************

All the notable changes added to the |NCS| v2.1.0 release are documented in this section.

Added
=====

* Added support for multi-instance transport.
  Each group can use a different transport instance, or groups can share a single transport instance.
* Added support for the nRF RPC protocol versioning.

Changes
=======

* Updated documentation.

nRF Connect SDK v2.0.0
**********************

All the notable changes added to the |NCS| v2.0.0 release are documented in this section.

Changes
=======

* Updated figures to follow style guidelines.
* Moved from TinyCBOR to zcbor.

nRF Connect SDK v1.9.0
**********************

No changes in this release.

nRF Connect SDK v1.8.0
**********************

All the notable changes added to the |NCS| v1.8.0 release are documented in this section.

Changes
=======

* Removed Latin abbreviations from the documentation.

nRF Connect SDK v1.7.0
**********************

All the notable changes added to the |NCS| v1.7.0 release are documented in this section.

Changes
=======

> * Used :kconfig: role as a reference to the Kconfig options.

Reviewer: We use :kconfig:option:`CONFIG_OPTION_NAME` for the formatting, but I could not find any instances within the RPC docs in nrfxlib.

Author: That's right, this change relates to the template header file.

nRF Connect SDK v1.6.0
**********************

All the notable changes added to the |NCS| v1.6.0 release are documented in this section.

Changes
=======

> * Renamed nrf_ernno.h to nrf_rpc_ernno.h.

Reviewer: ```suggestion
* Renamed :file:`nrf_ernno.h` to :file:`nrf_rpc_ernno.h`.
```
